### PR TITLE
Adding support for high priority queries to xlang transforms writing …

### PR DESCRIPTION
**Pull request by pabloem**
…to spanner

### CODE REVIEW

**Comment by pabloem** [comment]
this is needed because spanner autoscaling depends on high priority queries - so a spanner database can easily be overloaded by a beam pipeline and still not scale up

**Comment by pabloem** [comment]
is it still @johnjcasey ? : D

File: sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/spanner/SpannerTransformRegistrar.java [hunk 1]
```diff
@@ -341,6 +346,9 @@ public PTransform<PCollection<Row>, PDone> buildExternal(
               .withDatabaseId(configuration.databaseId)
               .withInstanceId(configuration.instanceId);
 
+      if (configuration.highPriority != null && configuration.highPriority) {
```
Comment (johnjcasey) [inline sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/spanner/SpannerTransformRegistrar.java:349]: is it worth defaulting to false just so you can avoid a null check?

Comment (damondouglas) [reply sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/spanner/SpannerTransformRegistrar.java:349]: I agree. Zero value should just be false. Thank you Pablo for doing this :-)

**Comment by pabloem** [comment]
done thanks!

**Comment by shunping** [comment]
LGTM

**Pull request merged**